index.html

```diff
@@ -952,47 +952,61 @@ <h3 property="schema:name">Namespaces</h3>
+                    <p>
+                      The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL
+                      NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and
+                      “OPTIONAL” are to be interpreted as described in
+                      <a href="https://tools.ietf.org/html/bcp14">BCP 14</a>
+                      [<cite><a class="bibref" href="#bib-rfc2119">RFC2119</a></cite
+                      >] [<cite
+                        ><a class="bibref" href="#bib-rfc8174">RFC8174</a></cite
+                      >] when, and only when, they appear in all capitals, as shown
+                      here.
                     </p>
-                    <pre
-                      about="#example"
-                      property="schema:description"
-                      typeof="fabio:Script"
-                    >
-                      <code>This is an example.</code>
-                    </pre>
-                  </figure>
 
-              </div>
-            </section>
+                    <p>
+                      Examples in this specification are introduced with the word "for example" or are set apart from the normative text with <code>class="example"</code>, like this:
+                    </p>
+                      <figure
+                        class="example listing"
+                        id="example"
+                        rel="schema:hasPart"
+                        resource="#example"
+                      >
+                        <p class="example-h">
+                          <span>Example 1</span>
+                        </p>
+                        <pre
+                          about="#example"
+                          property="schema:description"
+                          typeof="fabio:Script"
+                        >
+                          <code>This is an example.</code>
+                        </pre>
+                      </figure>
+
+                </div>
+              </section>
+
+              <section id="specification-category" inlist="" rel="schema:hasPart" resource="#specification-category" typeof="skos:ConceptScheme">
+                <h4 property="schema:name skos:prefLabel">Specification Category</h4>
+                <div datatype="rdf:HTML" property="schema:description">
+                  <p property="skos:definition">The <span about="" rel="spec:specificationCategory" resource="#specification-category">Solid WebID Profile</span> identifies the following <cite><a href="https://www.w3.org/TR/spec-variability/#spec-cat" rel="dcterms:subject" resource="spec:SpecificationCategory">Specification Category</a></cite> to distinguish the types of conformance: <span rel="skos:hasTopConcept" resource="spec:NotationSyntax">notation/syntax</span>, <span rel="skos:hasTopConcept" resource="spec:ProcessorBehaviour">processor behavior</span>, <span rel="skos:hasTopConcept" resource="spec:Protocol">protocol</span>.</p>
```

Comment:
Is this specification really covering protocol and processor behaviour as part of its requirements?

Reply:
Besides Notation/syntax I cannot identify what other category this migth fit in.